{"uid":"7b09a36956e13aa7","name":"Scenario Outline name: Verify that the Update API will be fails if recorded has future or invalid formatted date","fullName":"features.APITests.steps.test_update_steps#test_scenario_outline_name_verify_that_the_update_api_will_be_fails_if_recorded_has_future_or_invalid_formatted_date","historyId":"a3690769ab327e5823314f21edc7e9fe","time":{"start":1760009780697,"stop":1760009784706,"duration":4009},"description":"/home/runner/work/imms_fhir_api_automation/imms_fhir_api_automation/features/APITests/update.feature: Scenario Outline name: Verify that the Update API will be fails if recorded has future or invalid formatted date","descriptionHtml":"<p>/home/runner/work/imms_fhir_api_automation/imms_fhir_api_automation/features/APITests/update.feature: Scenario Outline name: Verify that the Update API will be fails if recorded has future or invalid formatted date</p>\n","status":"failed","statusMessage":"AssertionError: FAILURE: check bool(False): Expected Diagnostics: Validation errors: recorded must be a valid datetime in one of the following formats:- 'YYYY-MM-DD' — Full date only- 'YYYY-MM-DDThh:mm:ss%z' — Full date and time with timezone (e.g. +00:00 or +01:00)- 'YYYY-MM-DDThh:mm:ss.f%z' — Full date and time with milliseconds and timezone- Date must not be in the future., got Validation errors: recorded must be a valid datetime in one of the following formats:- 'YYYY-MM-DD' — Full date only- 'YYYY-MM-DDThh:mm:ss%z' — Full date and time with timezone (e.g. +00:00 or +01:00)- 'YYYY-MM-DDThh:mm:ss.f%z' — Full date and time with milliseconds and timezone-  Date must not be in the future.\nfeatures/APITests/steps/common_steps.py:205 in validateForbiddenAccess() -> validate_error_response(error_response, ErrorName[errorName].value, context.ImmsID)\nutilities/api_fhir_immunization_helper.py:104 in validate_error_response() -> check.is_true(\n\n------------------------------------------------------------\nFailed Checks: 1","statusTrace":"FAILURE: check bool(False): Expected Diagnostics: Validation errors: recorded must be a valid datetime in one of the following formats:- 'YYYY-MM-DD' — Full date only- 'YYYY-MM-DDThh:mm:ss%z' — Full date and time with timezone (e.g. +00:00 or +01:00)- 'YYYY-MM-DDThh:mm:ss.f%z' — Full date and time with milliseconds and timezone- Date must not be in the future., got Validation errors: recorded must be a valid datetime in one of the following formats:- 'YYYY-MM-DD' — Full date only- 'YYYY-MM-DDThh:mm:ss%z' — Full date and time with timezone (e.g. +00:00 or +01:00)- 'YYYY-MM-DDThh:mm:ss.f%z' — Full date and time with milliseconds and timezone-  Date must not be in the future.\nfeatures/APITests/steps/common_steps.py:205 in validateForbiddenAccess() -> validate_error_response(error_response, ErrorName[errorName].value, context.ImmsID)\nutilities/api_fhir_immunization_helper.py:104 in validate_error_response() -> check.is_true(\n\n------------------------------------------------------------\nFailed Checks: 1","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"setup_environment","time":{"start":1760009223884,"stop":1760009223884,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"global_context","time":{"start":1760009223886,"stop":1760009225414,"duration":1528},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"context","time":{"start":1760009780698,"stop":1760009782423,"duration":1725},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"_pytest_bdd_example","time":{"start":1760009817052,"stop":1760009817052,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"description":"/home/runner/work/imms_fhir_api_automation/imms_fhir_api_automation/features/APITests/update.feature: Scenario Outline name: Verify that the Update API will be fails if recorded has future or invalid formatted date","status":"failed","statusMessage":"AssertionError: FAILURE: check bool(False): Expected Diagnostics: Validation errors: recorded must be a valid datetime in one of the following formats:- 'YYYY-MM-DD' — Full date only- 'YYYY-MM-DDThh:mm:ss%z' — Full date and time with timezone (e.g. +00:00 or +01:00)- 'YYYY-MM-DDThh:mm:ss.f%z' — Full date and time with milliseconds and timezone- Date must not be in the future., got Validation errors: recorded must be a valid datetime in one of the following formats:- 'YYYY-MM-DD' — Full date only- 'YYYY-MM-DDThh:mm:ss%z' — Full date and time with timezone (e.g. +00:00 or +01:00)- 'YYYY-MM-DDThh:mm:ss.f%z' — Full date and time with milliseconds and timezone-  Date must not be in the future.\nfeatures/APITests/steps/common_steps.py:205 in validateForbiddenAccess() -> validate_error_response(error_response, ErrorName[errorName].value, context.ImmsID)\nutilities/api_fhir_immunization_helper.py:104 in validate_error_response() -> check.is_true(\n\n------------------------------------------------------------\nFailed Checks: 1","statusTrace":"FAILURE: check bool(False): Expected Diagnostics: Validation errors: recorded must be a valid datetime in one of the following formats:- 'YYYY-MM-DD' — Full date only- 'YYYY-MM-DDThh:mm:ss%z' — Full date and time with timezone (e.g. +00:00 or +01:00)- 'YYYY-MM-DDThh:mm:ss.f%z' — Full date and time with milliseconds and timezone- Date must not be in the future., got Validation errors: recorded must be a valid datetime in one of the following formats:- 'YYYY-MM-DD' — Full date only- 'YYYY-MM-DDThh:mm:ss%z' — Full date and time with timezone (e.g. +00:00 or +01:00)- 'YYYY-MM-DDThh:mm:ss.f%z' — Full date and time with milliseconds and timezone-  Date must not be in the future.\nfeatures/APITests/steps/common_steps.py:205 in validateForbiddenAccess() -> validate_error_response(error_response, ErrorName[errorName].value, context.ImmsID)\nutilities/api_fhir_immunization_helper.py:104 in validate_error_response() -> check.is_true(\n\n------------------------------------------------------------\nFailed Checks: 1","steps":[],"attachments":[{"uid":"cc59ccaa7691d438","name":"Step Passed: I have created a valid vaccination record","source":"cc59ccaa7691d438.txt","type":"text/plain","size":60},{"uid":"dc856ea6dd20c04d","name":"Step Passed: Send a update for Immunization event created with recorded being updated to 'nonexistent'","source":"dc856ea6dd20c04d.txt","type":"text/plain","size":108},{"uid":"b5dfad5e7728c7f7","name":"Step Passed: The request will be unsuccessful with the status code '400'","source":"b5dfad5e7728c7f7.txt","type":"text/plain","size":78},{"uid":"880ee68d2bd26c53","name":"Step Passed: The Response JSONs should contain correct error message for 'invalid_recorded'","source":"880ee68d2bd26c53.txt","type":"text/plain","size":97},{"uid":"c618e35e270aff62","name":"stdout","source":"c618e35e270aff62.txt","type":"text/plain","size":8914}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":5,"hasContent":true},"afterStages":[],"labels":[{"name":"epic","value":"Immunization Service"},{"name":"suite","value":"Update the immunization of a patient"},{"name":"feature","value":"Update the immunization of a patient"},{"name":"tag","value":"Update_Feature"},{"name":"tag","value":"vaccine_type_RSV"},{"name":"tag","value":"patient_id_Mod11_NHS"},{"name":"tag","value":"Delete_cleanUp"},{"name":"tag","value":"supplier_name_Postman_Auth"},{"name":"parentSuite","value":"features.APITests.steps"},{"name":"host","value":"runnervmwhb2z"},{"name":"thread","value":"2491-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"features.APITests.steps.test_update_steps"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"_pytest_bdd_example","value":"{'Date': 'nonexistent'}"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":["Update_Feature","supplier_name_Postman_Auth","vaccine_type_RSV","patient_id_Mod11_NHS","Delete_cleanUp"]},"source":"7b09a36956e13aa7.json","parameterValues":["{'Date': 'nonexistent'}"]}